repo_commits.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2014 The Gogs Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package gogs
+
+import (
+	"fmt"
+	"time"
+)
+
+type Signature struct {
```

Comment:
I think it is better to have a seperation between the webhooks and the commit models. It would be easier to extend them in the future.

And payload author  is not exactly the same as a commit signature. 

You could have commits from local users that does not have a user on gogs. And payload author lacks the when field. 

What do you think?